client/README.md

```diff
@@ -6,56 +6,58 @@ This extension provides support for Yseop Markup Language (YML).
-* `\beginStyle`: Styled text
-  ***or*** Text using a custom style
-* `\beginSynonym`: List of alternatives
-* `bold`: Bold text
-* `\case`: A case in a “switch” statement
-* `\choice`: An alternative in a list of synonyms
-* `complete`: Complete an instance with values
-* `\endOfLine`: Force end of line
-* `field`: Yml class attribute
-* `\foreach`: “For each” loop
-  ***or*** “For each” loop with assembly ID
-* `function`: Function
-  ***or*** Function with custom args
-  ***or*** Function with direct return
-* `granule`: Text granule
-* `instance`: Object instance
-* `interface`: Interface
-* `italic`: Italic text
-* `\nextItem`: An item in a static list
-* `\noContraction`: Prevent contraction
-* `\notEmptyString`: Make sure a string is not considered empty
-* `\nounPhrase`: Noun phrase
-* `numeralStyle`: Display style for numbers
-* `\recipientAction`: Verb using a second person subject
-  ***or*** Verb using a second person subject, with adjective
-* `round`: Number rounding
-* `\senderAction`: Verb using a first person subject
-  ***or*** Verb using a first person subject, with adjective
-* `\switch`: Handle values differently
-* `synonyms`: List of alternatives
-* `TextFunction`: Text function
-  ***or*** Text function with custom arguments
-* `textu`: Textualization attribute
-* `\thirdAction`: Verb using a third person subject
-* `\value`: Display a value
-* `\verb`: Verb
-
-[comment]: # (The snippet list can be updated using list-snippets.sh.)
-
+- `\amount`: Display amount
+- `\beginList`: Static List
+- `\beginQuote`: Text placed between quotes
+- `\beginStyle`: Styled text
+  **_or_** Text using a custom style
+- `\beginSynonym`: List of alternatives
+- `bold`: Bold text
+- `\case`: A case in a “switch” statement
+- `\choice`: An alternative in a list of synonyms
+- `complete`: Complete an instance with values
+- `\endOfLine`: Force end of line
+- `field`: Yml class attribute
+- `\foreach`: “For each” loop
+  **_or_** “For each” loop with assembly ID
+- `function`: Function
+  **_or_** Function with custom args
+  **_or_** Function with direct return
+- `granule`: Text granule
+- `instance`: Object instance
+- `interface`: Interface
+- `italic`: Italic text
+- `\nextItem`: An item in a static list
+- `\noContraction`: Prevent contraction
+- `\notEmptyString`: Make sure a string is not considered empty
+- `\nounPhrase`: Noun phrase
+- `numeralStyle`: Display style for numbers
+- `\recipientAction`: Verb using a second person subject
+  **_or_** Verb using a second person subject, with adjective
+- `round`: Number rounding
+- `\senderAction`: Verb using a first person subject
+  **_or_** Verb using a first person subject, with adjective
+- `\switch`: Handle values differently
+- `synonyms`: List of alternatives
+- `TextFunction`: Text function
+  **_or_** Text function with custom arguments
+- `textu`: Textualization attribute
+- `\thirdAction`: Verb using a third person subject
+- `\value`: Display a value
+- `\verb`: Verb
```

Comment:
Why change the bullets here and not in the script whose purpose is to generate the list?

Reply:
Automatic formatting, sorry; this was an unexpected change.